app/code/Academy/SmsSubscription/etc/db_schema.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<schema xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework:Setup/Declaration/Schema/etc/schema.xsd">
+    <table name="sms_subscription">
+        <column xsi:type="int" name="subscription_id" padding="11"
+                unsigned="true" nullable="false" identity="true" comment="SmsSubscription ID"/>
+        <column xsi:type="int" name="customer_id" unsigned="true" nullable="false" comment="Customer ID"/>
+        <column xsi:type="varchar" name="telephone" nullable="false" length="255" comment="Phone Number"/>
+        <column xsi:type="varchar" name="status" nullable="false" length="255" comment="Status"/>
+        <column xsi:type="timestamp" name="created_at" nullable="false" onCreate="true" comment="Created Datetime"/>
```

Comment:
In my opinion you don't need `onCreate` and `od_update`. Look at https://devdocs.magento.com/guides/v2.4/extension-dev-guide/declarative-schema/db-schema.html.